Makefile

```diff
@@ -19,7 +19,8 @@ CGO_ENABLED ?= 0
 NAMESPACE ?= shipwright-build
 
 # ko base image repository and options
-KO_DOCKER_REPO ?= $(IMAGE_BASE)
+IMAGE_HOST ?= ghcr.io
+IMAGE_NAMESPACE ?= shipwright-io/triggers
```

Comment:
Please leave the `KO_DOCKER_REPO` as a standalone variable, it get [exported as a regular environment variable using `.EXPORT_ALL_VARIABLES` target](https://github.com/shipwright-io/triggers/blob/3985967c8a670b2566cbc9f63c06fa32e199c642/Makefile#L48).

Thus, I suggest the following construction:

```Makefile
IMAGE_HOST ?= ghcr.io
IMAGE_NAMESPACE ?= shipwright-io/triggers
KO_DOCKER_REPO ?= $(IMAGE_HOST)/$(IMAGE_NAMESPACE)
```

The same approach takes place on `GOFLAGS`, no need to repeat yourself.

Reply:
fixed

Reply:
Thanks! 👍